### ISSUE TITLE
Crash on dragging

### ISSUE THREAD

**Issue by rodighiero** [davidfig/pixi-viewport#356]
I've to admit that it's a while I work on data and not with the interface itself, but one of the last updates broke the drag function and I don't understand why. As usual, I am working on a full-screen visualization whose code is pretty simple; the drag function is the one on the viewport. I include the code of the index.js and the build,

Thanks a lot,
Dario

https://github.com/rodighiero/weather-map/blob/main/src/index.js
https://rodighiero.github.io/weather-map/index.html

**Comment by rodighiero** [comment]
My problem is still unsolved, anyone who can help me please?

<img width="576" alt="Screen Shot 2021-11-25 at 4 21 03 PM" src="https://user-images.githubusercontent.com/2089004/143467160-f348d72d-c77a-4a2f-8d02-807d0055f09c.png">

**Comment by rodighiero** [comment]
It seems I have the same problem of #330 and #335, I downgrade to version 4.24 and the error disappeared.

**Comment by phschaad** [comment]
I am facing the same issue now, but a downgrade to 4.24 didn't fix it for me. What version of pixi are you using?

**Comment by rodighiero** [comment]
It's a while I am not coding, you can check my last configuration here https://github.com/rodighiero/weather-map/blob/main/package.json

**Comment by phschaad** [comment]
Thank you, I took your config as a starting point, though strangely enough it didn't work for me. For others arriving here via search, a configuration that worked for me:
- `pixi.js 6.1.3`
- `pixi-viewport 4.24`

Going any _higher_ with `pixi.js` _or_ `pixi-viewport` caused the problem to reappear. The issue seems to stem from the fact that `pixi-viewport` (probably after the typescript conversion) started relying on `@pixi/*` subpackages for dependencies.

**Comment by 9oelM** [comment]
Yeah I am experiencing the same issue. For me though, using the latest version of `pixi.js` worked for me.

```
"pixi-viewport": "4.24",
"pixi.js": "^6.4.2",
```
